generated/nidaqmx/nidaqmx.proto

```diff
@@ -3987,12 +5888,13 @@ enum PhysicalChannelInt32AttributeValues {
   PHYSICALCHANNEL_INT32_AI_MEASUREMENT_TYPE_TORQUE_BRIDGE = 15905;
   PHYSICALCHANNEL_INT32_AI_MEASUREMENT_TYPE_TEDS_SENSOR = 12531;
   PHYSICALCHANNEL_INT32_AI_MEASUREMENT_TYPE_CHARGE = 16105;
+  PHYSICALCHANNEL_INT32_AI_MEASUREMENT_TYPE_POWER = 16201;
   PHYSICALCHANNEL_INT32_AO_OUTPUT_CHANNEL_TYPE_VOLTAGE = 10322;
   PHYSICALCHANNEL_INT32_AO_OUTPUT_CHANNEL_TYPE_CURRENT = 10134;
   PHYSICALCHANNEL_INT32_AO_OUTPUT_CHANNEL_TYPE_FUNC_GEN = 14750;
-  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_10322 = 10322;
-  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_10134 = 10134;
-  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_12527 = 12527;
+  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_VOLTAGE = 10322;
+  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_CURRENT = 10134;
+  PHYSICALCHANNEL_INT32_AO_POWER_UP_OUTPUT_BEHAVIOR_HIGH_IMPEDANCE = 12527;
```

Comment:
This breaks source compatibility but it fixes a bug.

Reply:
Sure Brad. So, are we good with this change? or we should fix this as well?

Reply:
I think this change is okay. @astarche @reckenro do you agree?

Reply:
I think this change is okay as well. @maxxboehme , does this sound good to you?

@vigkre , if we do make this change can you make sure the `source-breaking` label to this PR?

Reply:
Checked with Maxx offline about this one as well and he agreed it was fine. Added that `source-breaking` label and marking this thread resolved.